circle.yml

```diff
@@ -99,8 +101,9 @@ jobs:
             docker-compose restart worker
             docker-compose exec web make -f Makefile-docker update_deps
             docker-compose restart web
+            sudo chown -R circleci:circleci .
```

Comment:
Add a comment why we do this.

Reply:
This comment only explains what it does, it doesn't explain why we have to change permissions. Also, you're not just changing permissions of `olympia.egg*` but recursively down all folders starting from the current one.